ssv/spectest/generate/main.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 	"github.com/bloxapp/ssv-spec/ssv/spectest/tests"
 	"github.com/bloxapp/ssv-spec/types"
+	comparable2 "github.com/bloxapp/ssv-spec/types/testingutils/comparable"
```

Comment:
comparable2 could be just comparable

Reply:
The thing is that `comparable` is a built in type in go.
So that's a bit confusing and that's why the IDEA automatically gives it a different name

There are places in the code where we call it `typescomparable` (because it is in `types` lib).
I think in another PR we need to choose a good name to use thorughout the project.

Any suggestions?

Reply:
since it compares states I guess `state` could be used as name, then using a function from the pkg could look like `state.Comparison`, which is nice. otherwise or if state is used, "statecomp", "compare/comp" just to clash with the stdlib.

Reply:
will do in other PR since it adds lots of changes adding to https://github.com/bloxapp/ssv-spec/issues/283